### ISSUE TITLE
Course File Archive

### ISSUE THREAD

**Issue by mickesv** [mickesv/LSRE-ReleasePlanningProject#15]
A Course shall have a file archive, so that material can be distributed to course participants.

Files in this archive may be shared over several instances of a course.

**Comment by saisrinivas8515** [comment]
Are the course participants allowed to share the files in the discussion?

**Comment by mickesv** [comment]
@saisrinivas8515 Yes. But not hotlink from outside the system. Add/clarify requirements to reflect this.

**Comment by akshaymagapu** [comment]
Sub-Requirements #30 #31
